### PR TITLE
sign those documents

### PR DESCRIPTION
**Pull request by srkirkland**
creates a document signing service which can impersonate a service account to send out shared templates, view doc status, download signed docs, and more

### CODE REVIEW

**Comment by acgetchell** [comment]
Awesome! The whole reason I was holding off on sending the CA&ES IT Security Guidelines for DocuSign-age until it could be tracked here!

File: Keas.Mvc/ClientApp/components/Documents/AssignDocument.tsx [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+  const { show, setShow } = props;
+
+  const renderTemplates = () => {
+    if (templates.length === 0) {
+      return (
+        <div>
+          No documents are available to send. Have a team administrator create
+          documents by following the instructions on{' '}
+          <a href='google.com'>our help page</a>
```
Comment (jSylvestre) [inline Keas.Mvc/ClientApp/components/Documents/AssignDocument.tsx:50]: @srkirkland We bought out google :)

File: Keas.Mvc/ClientApp/util/permissions.ts [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
   }
 
-  public static canViewAccess(r: string[]) {
+  public static canViewAccess(r: string[]): boolean {
     const permissionArray = ['AccessMaster', 'DepartmentalAdmin'];
     return r.some(a => permissionArray.includes(a));
   }
 
-  public static canViewPeople(r: string[]) {
+  public static canViewPeople(r: string[]): boolean {
```
Comment (jSylvestre) [inline Keas.Mvc/ClientApp/util/permissions.ts:17]: @srkirkland Need to Add DocumentMaster here?

File: Keas.Mvc/Controllers/DocumentSettingsController.cs [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+            return View(docSetting);
+        }
+        
+        [HttpPost]
+        public async Task<ActionResult> Delete(int id, TeamDocumentSetting docSetting)
+        {
+            var toDelete = await _context.TeamDocumentSettings.SingleAsync(t => t.Team.Slug == Team && t.Id == id);
+            _context.Remove(toDelete);
+            await _context.SaveChangesAsync();
+            Message = "Document Setting deleted.";
```
Comment (jSylvestre) [inline Keas.Mvc/Controllers/DocumentSettingsController.cs:88]: Maybe also say somewhere that deleting this just makes it unavailable for future use and does not affect existing ones?

File: Keas.Mvc/Views/DocumentSettings/Create.cshtml [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+      <div class="row">
+          <div class="col-md-8">
+            <button class="btn btn-link" type="button" data-toggle="collapse" data-target="#collapseInstructions" aria-expanded="false" aria-controls="collapseInstructions">
+                Show Instructions
+            </button>
+            </p>
+            <div class="collapse" id="collapseInstructions">
+                <div class="card card-body">
+                    <ol>
+                        <li>Login to <a href="https://app.docusign.com/home">docusign.com</a> and choose "Templates" from the top menu.  If you do not see that option, <a href="https://itcatalog.ucdavis.edu/service/docusign-pilot">request access here</a></li>
```
Comment (jSylvestre) [inline Keas.Mvc/Views/DocumentSettings/Create.cshtml:27]: Change so it create a new tab? Target _blank or something...

**Pull request merged**